packages/nextjs-demo/src/app/recipe/page.tsx

```diff
@@ -0,0 +1,19 @@
+import InputPrompt from '@/components/InputPrompt';
+import ResultContainer from '@/components/ResultContainer';
+import { RecipeGenerator } from '@/components/Recipe';
+
+export default function BasicCompletion({ searchParams }: { searchParams: any }) {
```

Comment:
maybe call this something else other than `BasicCompletion`?

Reply:
Good catch!